### PR TITLE
Update dns_filter.rst

### PR DESCRIPTION
**Pull request by cotosso**
adblock vs flashstart : choose just one

### CODE REVIEW

File: dns_filter.rst [hunk 1]
```diff
@@ -52,11 +52,13 @@ All listed IP addresses or networks here will not be affected by DNS filter.
 Block certain websites
 -------------------------
 
-.. highlight:: bash
-
 .. _block_website-section:
 
-Access to certain websites can also be restricted by manipulating their DNS resolution, enabling ``AdBlock`` and, optionally, activating the DNS query interception feature for LAN clients.
+If you need to block specific domains and FQDNs you can do it directly from the Flashstart configuration page.
```
Comment (gsanchietti) [inline dns_filter.rst:57]: Misspelled FlashStart. I'd also a link to the page of the manual.
```suggestion
If you need to block specific domains and FQDNs you can do it directly from the FlashStart configuration page.
```

File: dns_filter.rst [hunk 2]
```diff
@@ -52,11 +52,13 @@ All listed IP addresses or networks here will not be affected by DNS filter.
...
+
+If you don' have a subscription for Flashstart DNS Filter you can still make it directly on NethSecurity enabling ``AdBlock`` and, optionally, activating the DNS query interception feature for LAN clients.
```
Comment (gsanchietti) [inline dns_filter.rst:59]:
```suggestion
If you don' have a subscription for FlashStart DNS Filter you can still make it directly on NethSecurity enabling ``AdBlock`` and, optionally, activating the DNS query interception feature for LAN clients.
```

File: dns_filter.rst [hunk 3]
```diff
@@ -52,11 +52,13 @@ All listed IP addresses or networks here will not be affected by DNS filter.
...
+
+.. note:: Please use AdBlock to block browsing only if you are not already using the Flashstart service cause if used together, they may conflict.
```
Comment (gsanchietti) [inline dns_filter.rst:61]:
```suggestion
.. note:: Please use AdBlock to block browsing only if you are not already using the FlashStart service cause if used together, they may conflict.
```

File: dns_filter.rst (hunk 1 above)
Comment (cotosso) [reply dns_filter.rst:57]: Nothing in contrary , but in this document (dns_filter) flashstart is always written this way: "Flashstart".
If we change it here we need to change also in other places.

Comment (gsanchietti) [reply dns_filter.rst:57]: I'd say yes: we need to change it everywhere in the document.

Comment (cotosso) [reply dns_filter.rst]: I changed them and added a reference to find the right place in flashstart panel where to write them.

**Review by gsanchietti — APPROVED** [review]

**Pull request merged**